0051-helium-dao.md

```diff
@@ -50,6 +50,27 @@ There are three key drivers for proposing that DNP specific calculations are don
 
 This proposal impacts all current and future stakeholders in the Helium Community.
 
+# **Implementation**
+
+HIP 51 is to be implemented in three phases, each of which is contingent upon approval via governance across three separate voting procedures. Note that each vote, and therefore phase, must necessarily have code attached for consideration.
+
+The phases are defined as follows:
+
+1. Phase I (June 9, 2022) : As per HIP 53: Helium Mobile subDAO, MOBILE token is issued on the working Helium blockchain. 
+    1. The pre-mine amount specified in the Economics Overview section of the proposal is assigned to a multi-signature wallet held by the Helium Foundation.
+    2. The Helium Foundation then utilizes this reserve of MOBILE tokens to immediately perform market activities critical to the functioning of the Mobile network, including rewards for hotspots and community or business development efforts. 
+
+2. Phase II (August 1, 2022) : On the third anniversary of the Helium Network, the IOT token is issued on the working Helium blockchain as per HIP 52: IoT subDAO,
+    1. The specified emissions curve and treasury reserve redemption contract (one-way implementation of the market making curve) is deployed to the network such that core subDAO economic structures are in place. At this point, all HNT emissions are deposited to subDAO treasuries, and subnetwork hotspots, oracles, and ops funds receive DNTs.
```

Comment:
```suggestion
    1. The specified emissions curve and treasury reserve redemption contract (one-way implementation of the market making curve) is deployed to the network such that core subDAO economic structures are in place. This will allow MOBILE and IOT holders to redeem for HNT in their respective treasuries if they choose or continue to hold the DNT. At this point, all HNT emissions are deposited to subDAO treasuries, and subnetwork hotspots, oracles, and ops funds receive DNTs.
```

Reply:
More explicit here too